doc/api/thalireplicationmanager.md

```diff
@@ -0,0 +1,106 @@
+# The `ThaliReplicationManager` class
+
+The `ThaliReplicationManager` class handles database replication between devices, using [PouchDB](http://pouchdb.com/) and the Thali Cordova bridge `ThaliEmitter` class.
+
+## Usage
+
+This is the basic usage to start the replication manager.
+
+```js
+var PouchDB = require('pouchdb');
+var db = new PouchDB('dbname');
+
+var manager = new ThaliReplicationManager(db);
+
+manager.start(function (err) {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('Started the replication manager');
+
+    // Stop it after 5 seconds
+    setTimeout(function () {
+      manager.stop(function (err) {
+        if (err) {
+          console.log('err');
+        } else {
+          console.log('Stopped the replication manager');
+        }
+      })
+    }, 5000);
+  }
+});
+```
+
+## `ThaliReplicationManager(db)` constructor
```

Comment:
That's undefined behavior because I'm pretty sure it will just work to advertise and sync to any remote client.  I don't think it's necessary to point this out.